### PR TITLE
For #7345 - Update new shield icons, remove lottie

### PR DESCRIPTION
**Pull request by ekager**
Needs https://github.com/mozilla-mobile/android-components/pull/8701 before landing 

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by ekager** [comment]
Light, <24 API light_old

**Comment by ekager** [comment]
Dark <24 API

optimized_dark_small

**Comment by ekager** [comment]
24+ (with gradient), dark then light
ezgif com-video-to-gif

**Comment by ekager** [comment]
cc @AmyYLee to verify the 4 new animations

**Comment by ekager** [comment]
Chatted with Amy on Slack and updated the gradient angle and colors for 24+: 
Screen Shot 2020-10-15 at 9 58 55 AM
Screen Shot 2020-10-15 at 9 58 51 AM

**Comment by gabrielluong** [comment]
Sounds like we got the UX review from Amy. Dropping the label.

File: app/src/main/res/drawable-v24/shield_dark.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (gabrielluong) [inline app/src/main/res/drawable-v24/shield_dark.xml:1]: nit: license formatting.

We should maybe just fix this one day and just get the "This Source Code" start in the next line. Just putting this idea out there to see how it marinade. 

```
<?xml version="1.0" encoding="utf-8"?><!-- 
   - This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
```

**Review by gabrielluong — APPROVED** [review]

**Pull request merged**